src/game/scenes/vs.c

```diff
@@ -312,73 +319,83 @@ void vs_render(scene *scene) {
         char text[256];
         char money[16];
         fight_stats *fight_stats = &scene->gs->fight_stats;
+        static int force_redraw = 1;
+        if(force_redraw) {
+            // TODO: Find out why this caching doesn't work. The cache is built before the stats are correct.
```

Comment:
TODO: investigate this

Reply:
This problem can be seen when bringing up the in-game menu the second time through, it will show an empty spot where the "explanation" is on the N64 branch indicating that the text object wasn't correctly dirtied.

Reply:
There's a game menu on the 'vs' screen?

Reply:
Uhhh, that's interesting. I didn't realize I left this comment in the VS, it probably needs to be in arena somewhere.